### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Create_SAS URL_Screenreader : screen reader does not announce status message "Copied" when users click on copy button.

### ISSUE THREAD

**Issue by prasadaps** [microsoft/AzureStorageExplorer#3458]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG4.1.3;#NVDA;#JAWS;#Win32;#A11ySev2;#Benchmark;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10
Screen reader: JAWS(2020.1912.11ILM), NVDA Version:2020.2

### Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Blob gets uploaded, right click on blob and select "Get Shared Access Signature" control and hit enter. navigate to "Create" button and hit enter.
7. Start screen reader and navigate to copy button and press enter and check screen reader announce the status message copied.

### Actual:
When users navigate to copy button and press it the button change to copied showing users that the text is copied but same status message is not announce to screen reader users.

Note : This issue is observed with both JAWS and NVDA screen reader. 

### Expected:
The screen reader should announce the message that the text is copied.

### User Impact:
The screen reader users may not get the message that the data is copied.

Attachment
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-prphat_microsoft_com/ESFi2rxzjKVCo0e_BY7HUTUBq-mQZ3qNJoFeAAVhlfiX_g?e=aGudHZ

**Comment by JasonYeMSFT** [comment]
@prasadaps Could you please verify and close this issue if it is fixed?

**Comment by prasadaps** [comment]
@JasonYeMSFT I have verified the issue in application version 1.15.1. the issue is still reproducing.

**Comment by JasonYeMSFT** [comment]
@prasadaps Oh we haven't shipped a new version containing the fixes. Is there a way for you to access our latest build so you can test the fix without us actually shipping it?

**Comment by prasadaps** [comment]
@JasonYeMSFT I have checked the issue in application version 1.15.1 it was reproducing there. and had download this application from the link publicly available.
Also are you looking for me to check this bug is local build or test build. if so could you please provide me a link to download the build.

**Comment by prasadaps** [comment]
Verified the issue in latest build the issue is fixed hence closing the bug.

**Comment by 12-shweta** [comment]
**Github Tags:**#Closed;
